NDTensors/src/lib/GradedAxes/src/unitrangedual.jl

```diff
@@ -92,6 +105,9 @@ BlockArrays.blockaxes(a::UnitRangeDual) = blockaxes(nondual(a))
 BlockArrays.blockfirsts(a::UnitRangeDual) = label_dual.(blockfirsts(nondual(a)))
 BlockArrays.blocklasts(a::UnitRangeDual) = label_dual.(blocklasts(nondual(a)))
 BlockArrays.findblock(a::UnitRangeDual, index::Integer) = findblock(nondual(a), index)
+
+blocklabels(a::UnitRangeDual) = dual.(blocklabels(nondual(a)))
```

Comment:
I realized that once we have other types of dual ranges like `GradedUnitRangeDual` we probably want functions like this to be trait functions using an `IsDual` trait type based on the `isdual` function (say using [SimpleTraits.jl](https://github.com/mauro3/SimpleTraits.jl)). Then, this kind of function can work for any dual unit range type. Not needed for this PR, just commenting here as a reminder.

Reply:
I left a comment about this in #1531 so I'll close this.